src/sentry/migrations/0250_backfill_popularity.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.24 on 2021-11-15 22:47
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_popularity(apps, schema_editor):
+    SentryApp = apps.get_model("sentry", "SentryApp")
+
+    for sentry_app in RangeQuerySetWrapperWithProgressBar(SentryApp.objects.all()):
```

Comment:
I'm curious what the SQL looks like. This looks like we're going to do a bunch of serial update + commits as opposed to a commit at the end.

Reply:
yep, that's what this migration is meant to do. I mimicked the pattern I saw in https://github.com/getsentry/sentry/blob/master/src/sentry/migrations/0227_backfill_visits.py

This meant to fill in null values for a column I've recently added in https://github.com/getsentry/sentry/pull/29955

Reply:
Yeah, this is how we generally prefer to do this. If you were doing a very large update you could do some batching, but it's usually unnecessary.

We don't want to have one large transaction running in general because that can lock the entire table.

Reply:
That's good to know, thanks!